docs/plugins/verifier/cosign.md

```diff
@@ -192,16 +208,53 @@ kubectl run demo2 -n default --image=<insert test namespace image reference>
 kubectl logs deploy/ratify -n gatekeeper-system
 ```
 
-## Keyless Verification
+## Configuration
 
-This section outlines how to use `ratify` to verify the signatures signed using keyless signatures.
+### Kubernetes
 
-> [!WARNING]
-> Cosign keyless verification may result in verification timeout due to Fulcio and Rekor server latencies
+```yaml
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Verifier
+metadata:
+  name: verifier-cosign
+spec:
+  artifactTypes: application/vnd.dev.cosign.artifact.sig.v1+json
+  name: cosign
+  parameters:
+    trustPolicies: # OPTIONAL: [list], trust policies matching keys to scopes
+    - name: # REQUIRED: [string], trust policy name. MUST be unique across policies
+      version: # OPTIONAL: [string], trust policy schema version  
+      scopes: # REQUIRED: [list], string list of scopes
+      tLogVerify: # OPTIONAL: [boolean] enables/disables transparency log verification. default is 'true'
+      rekorURL: # OPTIONAL: [string] specifies a rekor URL pointing to a transparency log server. default is https://rekor.sigstore.dev
```

Comment:
nit: Just wondering if `Rekor` is better

Reply:
🤣 nvm, showing inconsistency in their own repo.

Ref: https://github.com/search?q=repo%3Asigstore%2Fdocs+rekor+server&type=code#:~:text=Cosign%20defaults%20to,url%60%20flag